cfme/tests/test_rest.py

```diff
@@ -26,7 +26,7 @@
     scope="module"
 )
 
-pytestmark = [pytest.mark.ignore_stream("5.2")]
+pytestmark = [pytest.mark.ignore_stream("5.3")]
```

Comment:
By doing this, you will effectively disable all REST tests for `5.3`. I'm not sure if we want to do that but at the same time I'm pretty sure there is not gonna be another `5.3.z` released so... I'm thinking: ok, let's do this, but if we really want to exclude `5.3` REST completely, we might want to remove all the other no-longer necessary code as well, e.g. `COLLECTIONS_IGNORED_53` and all the places where it's used. @psav? What do you think?